smart_card_connector_app/src/window-about-showing.js

```diff
@@ -46,7 +51,16 @@ const openAboutElement =
 function openAboutClickListener(e) {
   e.preventDefault();
 
-  chrome.app.window.create(ABOUT_WINDOW_URL, ABOUT_WINDOW_OPTIONS);
+  if (GSC.Packaging.MODE === GSC.Packaging.Mode.APP) {
+    chrome.app.window.create(ABOUT_WINDOW_URL, ABOUT_WINDOW_OPTIONS);
```

Comment:
nit: Actually, let's maybe delete the ABOUT_WINDOW_OPTIONS constant and just inline it here? It's a weird asymmetry that we have a constant for the App-specific API but no constant or the Extension-specific API.

Reply:
wouldn't inlining it be too crowded?

Reply:
A matter of taste I guess, but for this case - where the constant is only useful for one conditional branch but not for the other - I think it's fine.

If you don't like it, then what about doing it the other way round, i.e., extracting the chrome.windows.create's argument into a global constant? And clearing naming one as a app-specific and another as an extension-specific constant?